pkg/network/ebpf/c/prebuilt/tracer.c

```diff
@@ -495,7 +495,7 @@ int kprobe__udp_recvmsg(struct pt_regs* ctx) {
 SEC("kprobe/udp_recvmsg/pre_4_1_0")
 int kprobe__udp_recvmsg_pre_4_1_0(struct pt_regs* ctx) {
     struct sock* sk = (struct sock*)PT_REGS_PARM2(ctx);
-    struct msghdr* msg = (struct msghdr*) PT_REGS_PARM2(ctx);
+    struct msghdr* msg = (struct msghdr*) PT_REGS_PARM3(ctx);
```

Comment:
Why is this changing ? Was PT_REGS_PARM3 always wrong?

Reply:
I think you're looking at a diff between two commits in my branch. This line does not exist in master. I did have it wrong before in my branch since I had copied the lines from the regular udp_recvmsg probe above.

Reply:
never mind, the I was reading the wrong diff when I wrote this